applications/tari_app_grpc/proto/base_node.proto

```diff
@@ -264,3 +271,19 @@ message GetMempoolTransactionsRequest {
 message GetMempoolTransactionsResponse {
     Transaction transaction = 1;
 }
+
+message TransactionStateRequest {
+    Transaction transaction  = 1;
```

Comment:
Seems heavy to send a whole transaction in. Perhaps just the excess or a hash of a kernel?